lib/ansible/modules/messaging/rabbitmq/rabbitmq_policy.py

```diff
@@ -93,25 +101,55 @@ def __init__(self, module, name):
         self._node = module.params['node']
         self._rabbitmqctl = module.get_bin_path('rabbitmqctl', True)
 
-    def _exec(self, args, run_in_check_mode=False):
+        self._version = self._rabbit_version()
+
+    def _exec(self, args, run_in_check_mode=False, split_lines=True):
         if not self._module.check_mode or (self._module.check_mode and run_in_check_mode):
             cmd = [self._rabbitmqctl, '-q', '-n', self._node]
             args.insert(1, '-p')
```

Comment:
@puzan `rabbitmqctl status` doesn't support `vhost`.

As quick fix, I added `add_vhost=True` as default argument to `_exec` to have something like this :

```python
def _exec(self, args, run_in_check_mode=False, split_lines=True, add_vhost=True):
    .... some code here ....
    if add_vhost:
        args.insert(1, '-p')
        args.insert(2, self._vhost)
```

Reply:
I have added your solution.